modules/installer.nix

```diff
@@ -1,20 +1,28 @@
 { cfg, pkgs, lib, installation ? "system", ... }:
 
 let
+  gcroots =
```

Comment:
Could you maybe add a comment here wrt the risk of stateFile not getting GCed? 

Not a concern for now IMHO (see comments in thread), but I'd like to be reminded of this..

Reply:
Yeah makes sense, will add it